Dockerfile

```diff
@@ -41,7 +41,12 @@ RUN \
   apt-get -qq update && \
   apt-get -qq install apt-utils 2>/dev/null && \
   apt-get -qq dist-upgrade && \
+  echo "applying workaround for ubuntu/postfix bug described in https://github.com/docker-mailserver/docker-mailserver/issues/2023" && \
```

Comment:
```suggestion
  echo "Applying Dockerfile workaround detailed in: https://github.com/docker-mailserver/docker-mailserver/issues/2023#issuecomment-855326403" && \
```

Reply:
I like my version better

Reply:
I don't think it's an `ubuntu` bug specifically. But as the issue link has been updated to reference a relevant comment, it's good enough for me :+1: